packages/protocol/contracts/rrp/AirnodeRrp.sol

```diff
@@ -316,8 +317,25 @@ contract AirnodeRrp is
         )
     {
         delete requestIdToFulfillmentParameters[requestId];
-        // Failure is recorded so that it can be checked externally
-        requestWithIdHasFailed[requestId] = true;
-        emit FailedRequest(airnode, requestId);
+        emit FailedRequest(airnode, requestId, errorMessage);
```

Comment:
The only problem with doing that could be that the errorMessage is string and data is of type bytes